test/lib/collectionFactory.test.js

```diff
@@ -90,6 +90,34 @@ describe('Collection Factory', () => {
                 });
             });
         });
+
+        it('should create a Collection without pipelineIds', () => {
+            const dataWithoutPipelineIds = Object.assign({}, collectionData);
+
+            delete dataWithoutPipelineIds.pipelineIds;
+            datastore.save.resolves(dataWithoutPipelineIds);
```

Comment:
This should resolve pipelineIds as empty array right? That's what you put as save param